Apple-TV/PlaybackSettings.swift

```diff
@@ -0,0 +1,73 @@
+/*****************************************************************************
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Carola Nitz <caro # videolan.org>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import Foundation
+
+@objcMembers
+class PlaybackSettings: NSObject, Codable {
```

Comment:
Would you mind explaining the choice of making this Codable with the choice of JSON representation ?

Reply:
yes that way I only need to store the playbacksettings as one object and not every single property